### PR TITLE
Defect/2199 conf perm

### PR DESCRIPTION
**Pull request by rajeevyadav2**
add rep policy in /conf folder to provide read permissions to acs-commons-email-service user

### CODE REVIEW

**Comment by rajeevyadav2** [comment]
@HitmanInWis : I have created this new PR for 2199 issue. If it looks all good then please approve it.

**Comment by HitmanInWis** [comment]
Hi Rajeev, thanks for the refactor.  I will try to get to testing this in the coming days - unfortunately struggling to find the time but I do have this on my radar.

**Comment by HitmanInWis** [comment]
@rajeevyadav2 Can you please merge latest master into your branch to resolve conflicts?  I'd do it but I'm worried that doing so will cause me to steal the commit credit when this is merged.

**Comment by rajeevyadav2** [comment]
> @rajeevyadav2 Can you please merge latest master into your branch to resolve conflicts? I'd do it but I'm worried that doing so will cause me to steal the commit credit when this is merged.

@HitmanInWis : Done. :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 - #2206 fix sonar warnings; some package versions had to be increased
 - #2213 - Show/Hide Dialog Field TouchUI Widget: Fix hidden required field not disabled to save the dialog 
 - Fixed JcrJsonAdapter IllegalStateException when writing multi-valued JCR properties
+- #2199 Add read permission for acs-commons-email-service user in conf folder using rep policy
```
Comment (HitmanInWis) [inline CHANGELOG.md:24]: This should now be up in the `## Unreleased` section

**Comment by HitmanInWis** [comment]
Tested the PR - this is gtg once the CHANGELOG is fixed up.

File: CHANGELOG.md (hunk 1 above)
Comment (rajeevyadav2) [reply CHANGELOG.md]: @HitmanInWis : Moved it to unreleased.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 ## Unreleased ([details][unreleased changes details])
 <!-- Keep this up to date! After a release, change the tag name to the latest release -->
 [unreleased changes details]: https://github.com/Adobe-Consulting-Services/acs-aem-commons/compare/acs-aem-commons-4.3.2...HEAD
+- #2199 Add read permission for acs-commons-email-service user in conf folder using rep policy
```
Comment (HitmanInWis) [inline CHANGELOG.md:11]: Please add a `### Fixed` header above this, to match the format of the rest of the file.

**Comment by HitmanInWis** [comment]
Quick reminder @rajeevyadav2 in case you missed the requested change above.  The CHANGELOG is still not formatted correctly.

File: CHANGELOG.md (hunk 2 above)
Comment (davidjgonzalez) [reply CHANGELOG.md:11]: Ill resolve this on merge - thanks @rajeevyadav2 and @HitmanInWis !

**Comment by HitmanInWis** [comment]
Thanks @davidjgonzalez

**Pull request merged**